js/fileexplorer/shared/DirectoryExplorer.svelte

```diff
@@ -21,12 +20,14 @@
 	}>();
 	const tree = make_fs_store();
 
-	$: glob,
-		ignore_glob,
-		root,
+	const render_tree = (): void => {
```

Comment:
particulary nasty bug where all "object" types are considered changed in svelte due to how we re-render all components. This caused svelte to assume the server object changed on any  re-render, causing an `ls` server function to be fired on every component change, refreshing the tree. Took server out of the reactive statement to fix this.